build.sbt

```diff
@@ -66,6 +67,21 @@ lazy val cassandra = (project in file("cassandra"))
     IntegrationTest / testFrameworks += new TestFramework("weaver.framework.TestFramework")
   )
 
+lazy val `kafka-persistence` = (project in file("kafka-persistence"))
+  .dependsOn(core)
+  .configs(IntegrationTest)
+  .settings(commonSettings)
+  .settings(
+    name := "kafka-flow-kafka-persistence",
```

Comment:
Naming things is hard :)

Reply:
We, probably, should do something so `kafka-flow-cassandra` and `kafka-flow-kafka-persistence` have more similar names. I am having lack of fantasy today though :(

Reply:
@asavelyev01 How about `kafka-flow-persistence-cassandra` and `kafka-flow-persistence-kafka`?

Reply:
@rtar  I like the idea!

Reply:
I hope changing the name of already released module is not a big problem at this early stage.

Reply:
Renamed the modules